doc/tutorials/space_shooter/step_6.md

```diff
@@ -18,19 +17,18 @@ class SpaceShooterGame extends FlameGame
 }
 ```
 
-With that, Flame now will start to check if components has collided with each other. Next we need to
+With that, Flame now will start to check if components have collided with each other. Next we need to
 identify which components can cause collisions.
 
-In our case those are the `Bullet` and `Enemy` components and we need to add hitboxes to them.
+In our case those are the `Bullet` and `Enemy` components and we need to add `hitboxes` to them.
 
-A hitbox is nothing more than a defined part of the component's area that can hit
-other objects. Flame offers a collection of classes to define a hitbox, the simplest of them is
-the `RectangleHitbox`, which like the name implies will make a rectangular area as the component's
-hitbox.
+A `hitbox` is nothing more than a defined part of the component's area that can hit
```

Comment:
Removed. This depends on one's view, I suppose. I sometimes use backticks (or `<code>` tags, depending on your context) to highlight terms (as well as actual code - functions, var, classes, what have you). Since the doc did do the same here: https://github.com/flame-engine/flame/blob/main/doc/tutorials/space_shooter/step_6.md?plain=1#L54, I thought it made sense but, I respect your preference.